### PR TITLE
Add option to disable checking fragment identifier for a param named access_token

### LINKED ISSUE #189: Add option to disable access_token check
In `netlify-identity-widget/src/netlify-identity.js`, there is a `runRoutes` function that checks if the url hash matches various tokens—`invite_token`, `confirmation_token`, etc.

One of the tokens it checks for is `access_token`, which I believe is needed when you are using an external OAuth provider.

I am using the Netlify Identity Widget for authentication, but I am also doing some OAuth2 stuff outside of Netlify Identity (for authorization), which conflicts with the `access_token` check and results in a `index.js:2177 Error: Gotrue-js: Failed to parse tokenResponse claims` error.

Is it possible to add an init option that we pass through to `runRoutes` that disables the `access_token` check? 

I can put up a PR if this is something you would like to add.

Thanks!

### PR DESCRIPTION
**Pull request by BrandonNoad**
Fixes #189

This PR adds an init option to disable checking `location.hash` (the fragment identifier) for a param named `access_token`.

The option defaults to `false`, so there are no breaking changes.

I updated the README with a brief description of the new option.

Please enable "Hide whitespace changes".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,9 +126,10 @@ module API. Options include:
 
 ```js
 {
-  container: '#some-query-selector'; // container to attach to
-  APIUrl: 'https://www.example.com/.netlify/functions/identity'; // Absolute url to endpoint.  ONLY USE IN SPECIAL CASES!
-  namePlaceholder: 'some-placeholder-for-Name'; // custom placeholder for name input form
```
Comment (BrandonNoad) [inline README.md]: I replaced the `;` with `,` on these 3 lines.

**Comment by dstarrtrey** [comment]
I'd love to bump this PR! I've been trying to work around this issue for a while for a particular page also using Google OAuth.

**Comment by sw-yx** [comment]
im sorry i missed this for some reason i didnt get the notification the first time around. after reviewing this, i dont think its worth including, and suggest you either fork this package and own it, or use https://www.npmjs.com/package/patch-package to disable the check directly. im serious about it, its quite a good option!

thank you for the PR otherwise

**Comment by BrandonNoad** [comment]
@sw-yx No worries. I hadn't heard of patch-package, so thanks for suggesting it.

**Comment by sw-yx** [comment]
its freaking amazing.

**Pull request closed without merging**